dipy/data/fetcher.py

```diff
@@ -611,33 +635,22 @@ def read_mni_template(contrast="T2"):
     >>> T2_nifti = read_mni_template("T2") # doctest: +SKIP
     Get both files in this order:
     >>> T1_nifti, T2_nifti = read_mni_template(["T1", "T2"]) # doctest: +SKIP
-
     """
-    folder = pjoin(dipy_home, 'mni_template')
+    files, folder = fetch_mni_template()
     file_dict = {"T1":pjoin(folder, 'mni_icbm152_t1_tal_nlin_asym_09a.nii'),
```

Comment:
Here's the license for the IIT atlas : http://www.iit.edu/~mri/Downloads.html

It would be a huge help if the atlas were under a more open license such as PDDL .